### DISCUSSION TITLE
UTM Crash on boot and subsequently crashes even after uninstall

### DISCUSSION THREAD

**Discussion by synackal** [utmapp/UTM#6138] [UTM Help (macOS)]
My machine became unresponsive and I had to force reboot. When I came back and attempted to launch UTM I received the crash found below. I tried uninstalling UTM (I don't know all the associated dependencies it lays down but tried my best via lsof) and then reinstalling and I am still receiving this crash.

* UTM Version: 4.4.4 (93)
* macOS Version: 14.3 
* Mac Chip: Apple M2 Max

[UTM-2023-12-19-110126.txt](https://github.com/utmapp/UTM/files/13719502/UTM-2023-12-19-110126.txt)

**Comment by osy** [comment]
I think your installation is corrupted. Download UTM.dmg from the releases and replace UTM.app
